views/rosterview/login.hbs

```diff
@@ -2,13 +2,22 @@
 
 <form action="/user/login" method="post">
     <div class="col s12">
-        <input type="email" name="email" class="text col s4 offset-m4" placeholder="Email">
-        <input type="password" name="password" class="text col s4 offset-m4" placeholder="Password">
-        <div class="col s3 offset-m6">
-            <button class="btn waves-effect waves-light col s7" type="submit">Login
-                <i class="material-icons right">login</i/>
-            </button>
-        </div>
+        <input type="email" name="email" class="text col s4 offset-m4" placeholder="Email" required>
+        <input type="password" name="password" class="text col s4 offset-m4" placeholder="Password" required>
+    </div>
+    <div class="col s12">
+        <button class="btn waves-effect waves-light col s4 offset-m4" type="submit">Login
+            <i class="material-icons right">login</i/>
+        </button>
     </div>
 </form>
 
+{{#if errors }}
+<script>
+    var error = "Opps, {{errors}}"
+    document.addEventListener('DOMContentLoaded', (event) => {
+        M.toast({html: error})
+    });
+</script>
```

Comment:
There are lots of ways to do this but seemed easy just to add the script to the login and let handlebars decide if to have it execute or not dependent on the presence of the errors.